src/plugins/local/components/TableHeadingCellContainer.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { compose, withHandlers, mapProps } from 'recompose';
+import { sortPropertyByIdSelector } from '../../../selectors/dataSelectors';
+import { setSortColumn } from '../../../actions';
+
+function setSortProperties({setSortColumn, sortProperty, columnId}) {
```

Comment:
Will eventually have jsdoc but this basically wraps the event in a closure and sets up the sortProperty object for this column and passes it along. The sort property objects are what makes multiSort / multiDirection (aka name could be sort ascending as the primary sort and state could be secondary sort descending -- this is not totally reflected in this PR but keeping the sortProperties setup this way makes that a lot easier).